library/core/src/mem/maybe_uninit.rs

```diff
@@ -851,4 +851,154 @@ impl<T> MaybeUninit<T> {
     pub fn slice_as_mut_ptr(this: &mut [MaybeUninit<T>]) -> *mut T {
         this.as_mut_ptr() as *mut T
     }
+
+    /// Copies the elements from `src` to `this`, returning a mutable reference to the now initalized contents of `this`.
+    /// Any already initalized elements will not be dropped.
```

Comment:
```suggestion
```
This sentence is useless because `T: Copy` always implies that `mem::drop::<T>` does nothing.